src/internal_events/mod.rs

```diff
@@ -89,6 +90,7 @@ mod mongodb_metrics;
 #[cfg(feature = "sources-nginx_metrics")]
 mod nginx_metrics;
 mod open;
+#[cfg(feature = "transforms-log_to_metric")]
```

Comment:
Hey @jszwedko thanks for the heads-up.
That one was tricky indeed. Can you tell me which component required that parser code?
I did my best to try to find it and couldn't. Also, how come the CI didn't fail for this PR if that was the case?

I had to add that flag to the parser because when compiling Vector with minimal features, I started to get parser import unused errors. I think the correct thing to do is to restore that flag and add the other features that also use it.

Reply:
Aha, yeah, I see, we will get dead code warnings. I think it may be safer to just revert this temporarily then and let you reintroduce it with the additional flag handling. Let me do that. You can see the flags that failed to compile in isolation here: https://github.com/vectordotdev/vector/actions/runs/5908210923/job/16027341665  (e.g. `sources-kubernetes_logs`). As you noted, you can see these locally via `make check-component-features`. Just a warning that this check takes a _long_ time 😓 since it tries to compile each feature individually. We can also trigger CI to run it on a new PR from you if that helps.

Reply:
Thanks for the heads-up that it's an expensive operation.
I will check out the job output to determine the dependencies and prepare a new PR shortly.
I think it would be a good idea to trigger a CI run for that PR to verify. The second time is the charm!